src/main/resources/application.yml

```diff
@@ -27,6 +27,7 @@ application:
     form-updated: ${FORM_UPDATED_QUEUE}
     gmc-rejected: ${GMC_REJECTED_QUEUE}
     gmc-updated: ${GMC_UPDATED_QUEUE}
+    ltft-updated:
```

Comment:
Can add the parameter for it with `${xxx}`,
and also add it as the secrets in the `task-detinition-template.json`, so it will link with the parameter name of the queue in the AWS Parameter Store.